packages/controller-utils/src/constants.ts

```diff
@@ -110,3 +110,9 @@ export enum ApprovalType {
   Unlock = 'unlock',
   ConnectAccounts = 'connect_accounts',
 }
+
+export const NETWORK_ID_TO_ETHERS_NETWORK_NAME_MAP = {
+  [NetworksChainId.goerli]: NetworkType.goerli,
```

Comment:
Nit: The name of this map suggests that they keys would be network IDs, but these are chain IDs. They happen to be the same at the moment, but that may not be true if we change the `chainId` format or add a network where they differ.

Instead I'd recommend creating an enum of network IDs, and using those here instead.